python/dgl/dataloading2/graph_storage/csc_sampling_graph.py

```diff
@@ -0,0 +1,309 @@
+"""Csc format sampling graph."""
+# pylint: disable= invalid-name
+from typing import List, Optional, Tuple
+
+import torch
+
+from .base import Graph
+
+HeteroInfo = Tuple[List[str], List[str], torch.tensor, torch.tensor]
+
+
+class CSCSamplingGraph(Graph):
```

Comment:
Missing methods to get the string name of node/edge types. Edge types should further be string triplets.

Reply:
Docstring of non-straightforward methods should have examples in docstring.